query-node/mappings/src/workingGroups.ts

```diff
@@ -714,18 +714,18 @@ export async function workingGroups_StatusTextChanged({ store, event }: EventCon
 }
 
 export async function workingGroups_LeadRemarked({ store, event }: EventContext & StoreContext): Promise<void> {
-  const [metadataByte] = new WorkingGroup_LeadRemarkedEvent_V1001(event).params
+  const [metadataByte, ] = new WorkingGroup_LeadRemarkedEvent_V1001(event).params
   const group = await getWorkingGroup(store, event)
 
   const metadata = deserializeMetadata(RemarkMetadataAction, metadataByte)
   if (metadata?.moderatePost) {
     if (group.name !== 'forumWorkingGroup') {
       return invalidMetadata(`The ${group.name} is incompatible with the remarked moderatePost`)
     }
-    const { postId, rationale } = metadata.moderatePost
+    const { postId, rationale, validator } = metadata.moderatePost
     const actor = await getWorkingGroupLead(store, group.name)
 
-    await moderatePost(store, event, 'leadRemark', postId, actor, rationale)
+    await moderatePost(store, event, 'leadRemark', postId, actor, rationale, validator)
   } else {
```

Comment:
That's not it. The idea is to add one more case to this function. ATM there is just one case which is `if (metadata?.moderatePost) { ... }` otherwise the mapping returns an `invalidMetadata` errror. So in addition to the `metadata?.moderatePost` there should be a e.g `metadata?.verifyValidator` case.